twittermapping.json

```diff
@@ -106,4 +106,6 @@
   , "simonswain": "simon_swain"
   , "vaskas": "vaskas"
   , "sebarmeli": "sebarmeli"
+  , "mauricebutler": "ButlerMaurice"
```

Comment:
case sensitive, you're `MauriceButler` so it's not being picked up, try again @MauriceButler